tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Features/Ods Organisations GetByOdsCode 3536.feature

```diff
@@ -84,4 +84,4 @@ Scenario: 8. Service Failure
 		| PostmanPat@email.com | An0therPa$$w0rd | Organisation |
 	And Ods API is down
 	When a GET request is made for an Ods organisation with code B1G
-	Then a response with status code 500 is returned
+	Then a response with status code 500 is returned
```

Comment:
File should end with a newline to keep git happy.

Also, can you untabify the file? (Select All then Edit->Advanced->Untabify Selected Lines in VS.)

Reply:
Line endings seem to be a recurring theme of my PRs. I'll try watch out for this. 😅 
Thanks